Assets/Razgriz/VRCFTGenerator/VRCFTGenerator.cs

```diff
@@ -330,11 +330,12 @@ private void CreateFTBlendshapeController()
                         childMotion = parameterDirectBlendTreePositive;
                     }
 
-                    binarySumTopLevelChildMotions[pr] = new ChildMotion {motion = childMotion, directBlendParameter = binarySumTopLevelNormalizerParam.Name, timeScale = 1.0f, threshold = 0.0f};
+                    binarySumTopLevelChildMotions.Add(new ChildMotion {motion = childMotion, directBlendParameter = binarySumTopLevelNormalizerParam.Name, timeScale = 1.0f, threshold = 0.0f});
 
                     pr++;
                 }
 
+//                pr = 0;
                 foreach (string param in parseLayers)
```

Comment:
With this change, this foreach adds extra children. To fix, change to 

`foreach (string param in selectedFloatParams)`